styles/components/_autocomplete.scss

```diff
@@ -0,0 +1,40 @@
+// ==================================================================
+// Autocomplete
+// ==================================================================
+
+////
+/// @group autocomplete
+////
+.sprk-c-Autocomplete__results {
+  list-style-type: $sprk-autocomplete-results-list-style-type;
+  background-color: $sprk-white;
```

Comment:
This should be an autocomplete var that is then set to $sprk-white so that away it is configurable.